ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```
@@ -76,9 +76,14 @@ declare -a MIRRORED_IMAGES=(
 )
 
 function run-oc-image-mirror() {
-  oc image mirror -f /tmp/mirror --keep-manifest-list --registry-config ${DS_WORKING_DIR}/pull_secret.json || return 1
+  local env_prefix=()
+  if [[ "${DISABLE_HTTP2_IMAGE_MIRRORING:-}" == "1" ]]; then
+    env_prefix=(env DISABLE_HTTP2=1)
+  fi
+
+  "${env_prefix[@]}" oc image mirror -f /tmp/mirror --keep-manifest-list --registry-config ${DS_WORKING_DIR}/pull_secret.json || return 1
```

Shellcheck (0.11.0)

[warning] env_prefix is referenced but not assigned.

(SC2154)